src/main/java/ca/bc/gov/backendstartapi/util/ObjectUtil.java

```diff
@@ -0,0 +1,47 @@
+package ca.bc.gov.backendstartapi.util;
```

Comment:
I believe we should remove the cases dealing with arrays or collections if we agree to set the rule of **not passing or returning `null` instead of an array or `Collection`**.

Is there any practical meaning `null` could have that empty instances of those could not? If we take the logical meaning of collections to be "all the elements that meet a set of conditions" and that of arrays and indexed collections to be "an ordered representation of all the elements that meet a set of conditions", I believe we don't.

[Paradoxes aside](https://plato.stanford.edu/entries/russell-paradox/), we should always have an answer for that: if there are any, we'll have a collection with them; if there is none, we'll have an empty collection. `null` *can* be used when we don't have an answer, but I believe that for our purposes such cases would be better represented with errors and exceptions.

Reply:
Agreed.